client/src/Authentication/RoleBasedRoutes.js

```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from 'react';
 import { Outlet, Navigate } from 'react-router-dom'
-import { getUserType } from '../Utils/UserType';
+import { getUserType } from '../helpers/UserType.js';
```

Comment:
Remove the extension, this will introduce a bug
```suggestion
import { getUserType } from '../helpers/UserType';
```